cmd/scw/testdata/test-all-usage-ipfs-name-usage.golden

```diff
@@ -0,0 +1,15 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+This is the PKI namespace, where the private key is used to publish (sign) a record.
+
+USAGE:
+  scw ipfs name
```

Comment:
What are the available verbs for this resource?

Reply:
create / export-key / import-key / list / delete